packages/xy-chart/src/utils/chartUtils.js

```diff
@@ -77,6 +77,27 @@ export function propOrFallback(props, propName, fallback) {
   return props && isDefined(props[propName]) ? props[propName] : fallback;
 }
 
+export function scaleInvert(scale, value) {
+  if (!scale.invert) {
```

Comment:
could we just add a comment here that band scales don't have the `invert` method?